bashlex/tokenizer.py

```diff
@@ -968,12 +968,15 @@ def handledollarword():
 
     def _is_assignment(self, value, iscompassign):
         c = value[0]
+        
+        def legalvariablestarter(x):
+            return x.isalpha()
```

Comment:
missing `or x == '_'` in here since `_x` is a legal variable name. please add a test.

Reply:
I fixed this already.